pages/app/dashboard.tsx

```diff
@@ -1,13 +1,84 @@
-import React from "react";
+import React, { useState } from "react";
+import Image from "next/image";
 import { useTranslations } from "next-intl";
+import Navbar from "@/resources/containers/nav";
+import SideBar from "@/resources/containers/sidebar";
+import dash_styles from "@/styles/pages/dashboard.module.css";
 import { getStaticPropsWithTranslations } from "@/modules/lang/props";
 
 export const getStaticProps = getStaticPropsWithTranslations;
 
 const Dashboard: React.FC = () => {
   const t = useTranslations("");
+  const [sidebarOpen, setSidebarOpen] = useState(false);
 
-  return <h1>{t("Dashboard")}</h1>;
+  const sideBarItems = [
+    { icon: "/recordings.svg", label: "Recordings" },
+    { icon: "/history.svg", label: "Historical" },
+  ];
+
+  const toggleSidebar = () => {
+    setSidebarOpen(!sidebarOpen);
+  };
+
+  return (
+    <div className={dash_styles.container}>
+      <Navbar toggleSidebar={toggleSidebar} />
+      <div className={dash_styles.contentWrapper}>
+        <SideBar
+          _activeTab={"Recordings"}
+          isOpen={sidebarOpen}
+          closeSidebar={() => setSidebarOpen(false)}
+          sideBarItems={sideBarItems}
+        />
+        <main className={dash_styles.mainContent}>
+          <h2 className={dash_styles.h2}>
```

Comment:
All the content of the main shall go in the modules folder

`modules/record/` 

Here is the frame

````
<Main>
 <TopBar>
  <SideMenu>
   <Recordings> <--- Later shall be dynamic, now is ok
   
</Main>

Actially everuthinb but recordings shall go in the Layout, and in this page just load the Recordings Component